src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1671,6 +1671,54 @@ paths:
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
 
+  /state/account/page/resource-preferences:
+    post:
+      operationId: AccountResourcePreferencesPage
+      summary: Get Account resource preferences
+      description: |
+        Returns paginable collection of resource preference rules for given account.
+      tags:
+        - State
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/StateAccountResourcePreferencesPageRequest"
+      responses:
+        "200":
+          description: Account resource preferences page
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/StateAccountResourcePreferencesPageResponse"
```

Comment:
Another minor note / question: do we want to name those types after the path or keep this convention? The "page" segment occurs earlier in the URL path compared to the proposed type name.

Reply:
I'd keep convention that we have in other places, so if I understand that's what I've committed here.